doc/releases/release-notes-2.3.rst

```diff
@@ -18,6 +18,18 @@ Security Vulnerability Related
 
 No security vulnerabilities received.
 
+API Changes
+***********
+
+Deprecated in this release
+==========================
+
+Stable API changes in this release
+==================================
+
+Removed APIs in this release
+============================
+
```

Comment:
what does this change have to do with `ext: lib: Move fnmatch to lib/`?

Reply:
ooops, fixed @MaureenHelm, thanks!